onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.h

```diff
@@ -5,6 +5,7 @@
 
 #if !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)
 
```

Comment:
Note: The modifications in this file disable QDQ fusions in QDQ::selectorActionTransformer for int4 QDQ node groups.